core/src/main/scala/com/salesforce/op/filters/FeatureDistribution.scala

```diff
@@ -142,6 +143,13 @@ case class FeatureDistribution
 private[op] object FeatureDistribution {
 
   val MaxBins = 100000
+  val AvgBinValue = 5000
+  val MaxTokenLowerLimit = 10
+  val getBins = (sum: Summary, bins: Int) => {
```

Comment:
let's add move this method to Summary class:
```
private[op] case class Summary(min: Double, max: Double) {
   
   // TODO: docs
   def bins(bins: Int): Int = {
       if (sum.max < MaxTokenLowerLimit) bins
       else math.min(math.max(bins, sum.sum / AvgBinValue), MaxBins).toInt
   }

}

Reply:
fixed!